Apps/Sandcastle/index.html

```diff
@@ -21,6 +21,7 @@
     <script src="jsHintOptions.js"></script>
     <script src="gallery/gallery-index.js"></script>
     <script src="CesiumSandcastle.js"></script>
+    <script src="../../Source/Cesium.js"></script>
```

Comment:
I have some minor concerns that including Cesium here will slow down Sandcastle loading even more than it already is.  That's a ton of files to retrieve up front.  We should probably do some smokescreen testing with an empty cache and see if it's noticeably worse.

As an aside, we really need to look into a having Sandcastle use a build version of Cesium to avoid these issues, but that's out of scope.)